Cargo.toml

```diff
@@ -82,3 +82,4 @@ icrc-ledger-types = "0.1.0"
 ic-ledger-types = "0.8"
 dfinity-stable-structures = { package = "ic-stable-structures", version = "0.6" }
 ic-test-state-machine-client = "3"
```

Comment:
To be removed. Please remove it in the scope of this task or create a dedicated one

Reply:
Will do

Reply:
Done

Reply:
Done